luigi/target.py

```diff
@@ -207,7 +207,8 @@ def __init__(self, path):
 
         :param str path: the path associated with this FileSystemTarget.
         """
-        self.path = path
+        # cast to str path objects like pathlib.PosixPath and py._path.local.LocalPath
```

Comment:
Can we reword this in-line comment for clarity? `cast to str to allow path to be objects like pathlib....`